src/riscv/i_sound.c

```diff
@@ -1,7 +1,7 @@
 /*
  * i_sound.c
  *
- * Dummy sound code
+ * Sound system support code
  *
  * Copyright (C) 2021 Sylvain Munaut
```

Comment:
As this file is no longer an implementation of a dummy sound device, you can confidently change the copyright notice. Please ensure to specify your full formal name in the notice.